cmake-modules/AddTestCMocka.cmake

```diff
@@ -1,39 +1,74 @@
+#
+# Copyright (c) 2007      Daniel Gollub <dgollub@suse.de>
+# Copyright (c) 2007-2018 Andreas Schneider <asn@cryptomilk.org>
+# Copyright (c) 2018      Anderson Toshiyuki Sasaki <ansasaki@redhat.com>
+#
+# Redistribution and use is allowed according to the terms of the BSD license.
+# For details see the accompanying COPYING-CMAKE-SCRIPTS file.
```

Comment:
We don't have such a file. Consider removing this line.

```suggestion
```

Reply:
However, I think we have to have it. For our own modules